v3/how-to-guides/01-basics/c-constants/index.mdx

```diff
@@ -5,138 +5,141 @@ keywords: basics, runtime, configurable constants
 version: '3.0'
 section: how to guides
 category: basics
+difficulty: 1
 ---
 
-Declaring a constant value in a runtime is a useful tool to either define fixed values or define values that change dynamically according to some factor. This guide shows you how to create pallet constants that are used to reset a \`u32\` value in storage. This value, we'll call \`SingleValue\`, can also be modified using a method called \`add_value\`.
+Declaring a constant value in a runtime is a useful tool to either define fixed values or define values that change dynamically according to some factor.
```

Comment:
Not sure why we're using `\` here...

Reply:
this was an artifact from the old `<Message>` mdx item